lib/features/thread/presentation/widgets/dismissible_widget.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:core/presentation/extensions/color_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+import 'package:tmail_ui_user/features/thread/presentation/mixin/base_email_item_tile.dart';
+
+class DismissibleWidget<T> extends StatelessWidget with BaseEmailItemTile {
```

Comment:
why we need this widget?

Reply:
this widget is the second tile if we slide right or left it take the two
icon and item and the function of Dismissed I separate in a new file to make the code cleaner